### ISSUE TITLE
docs build fails

### ISSUE THREAD

**Issue by dfguerrerom** [openforis/sepal_ui#917]
There is an error with the documentation build as it can be seen here:
https://readthedocs.org/projects/sepal-ui/builds/24339120/

```bash

Could not import extension jupyter_sphinx (exception: lxml.html.clean module is now a separate project lxml_html_clean.

```

**Comment by 12rambau** [comment]
arf this seems to be related to jupyter_sphinx itself that I am supposed to maintain as well

**Comment by dfguerrerom** [comment]
fixed with 965da7062c5f97cf0a25db114c329e8f023cb594

**Comment by dfguerrerom** [comment]
Actually it continuous failing in readthedocs... I have opened a new PR to test it: https://github.com/openforis/sepal_ui/pull/930

**Comment by dfguerrerom** [comment]
The error is not the same as the previous, that one was fixed with 965da70, the issue now is:
```console
  Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/sepal-ui/envs/930/lib/python3.10/site-packages/m2r2.py", line 611, in run
    path = nodes.reprunicode(path)
AttributeError: module 'docutils.nodes' has no attribute 'reprunicode'
The full traceback has been saved in /tmp/sphinx-err-n3qk6yyy.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```    

To see the fulltraceback: https://readthedocs.org/projects/sepal-ui/builds/25238989/

**Comment by 12rambau** [comment]
m2r2 have not been updated for a year , incompatibility with latest docutils is to be expected. Can you trace back to where the m2r2 package is called ?
